src/test/java/com/careerfair/q/service/employee/implementation/EmployeeServiceTest.java

```diff
@@ -65,24 +65,24 @@ public void testGetEmployeeWithEmail() {
 
     @Test
     public void testUpdateEmployeeName() {
-        UpdateEmployeeRequest updatedNameRequest = new UpdateEmployeeRequest();
-        updatedNameRequest.setName("n2");
-        updatedNameRequest.setCompanyId("c1");
-        updatedNameRequest.setRole(Role.SWE);
-        updatedNameRequest.setBio("b1");
-        updatedNameRequest.setEmail("e1@c1.com");
+        UpdateEmployeeRequest updateNameRequest = new UpdateEmployeeRequest();
+        updateNameRequest.setName("n2");
+        updateNameRequest.setCompanyId("c1");
+        updateNameRequest.setRole(Role.SWE);
+        updateNameRequest.setBio("b1");
+        updateNameRequest.setEmail("e1@c1.com");
```

Comment:
filter into private method

Reply:
employee service, will make new branch and do it